README.md

```diff
@@ -99,24 +99,41 @@ on:
 
 jobs:
   cache-and-install:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     steps:
-      build:
```

Comment:
A valid action should always have a job name.

Reply:
Hi. I noticed. The job name is `cache-and-install`. In that example was an error.

Reply:
Because is under `steps`.